### PR TITLE
Grammar Refined

### PR DESCRIPTION
**Pull request by SauveJeanLuc**
This version contains corrections of some grammatical mistakes which were committed throughout the project.

### CODE REVIEW

**Comment by mdevils** [comment]
@SauveJeanLuc great job, thank you. Left 1 comment.

**Comment by SauveJeanLuc** [comment]
Which comment @mdevils ?

**Comment by mdevils** [comment]
@SauveJeanLuc forgot to publish it. Here you go.

File: src/exercises/1/index.ts [hunk 1]
```diff
@@ -62,12 +62,12 @@ Brief UI guide:
 
 Intro:
 
-    We are starting a small community of users. For performance
+    We are starting a small community of users. For perfomance
```
Comment (mdevils) [inline src/exercises/1/index.ts:65]: Same as above ^

File: src/exercises/1/index.solution.ts [hunk 2]
```diff
@@ -62,12 +62,12 @@ Brief UI guide:
 
 Intro:
 
-    We are starting a small community of users. For performance
-    reasons we have decided to store all users right in the code.
+    We are starting a small community of users. For perfomance
```
[the remark this answers is not in the document]
Comment (SauveJeanLuc) [reply src/exercises/1/index.solution.ts]: Fixed

**Comment by mdevils** [comment]
@SauveJeanLuc Great, thanks! If you rebase, I'll merge.

**Pull request merged**